image_width: 640
image_height: 480
num_gpu: -1                 # The number of GPU devices to use. If negative, it 
                              # will use all the available GPUs in your machine.
num_gpu_start: 0            # GPU device start number
flag_model_pose: "COCO"          # Model to be used (e.g. COCO, MPI, MPI_4_layers)
flag_net_resolution: "-1x368"    # Multiples of 16. If it is increased, the accuracy potentially increases. If it is
                              # decreased, the speed increases. For maximum speed-accuracy balance, it should keep the
                              # closest aspect ratio possible to the images or videos to be processed. Using `-1` in
                              # any of the dimensions, OP will choose the optimal aspect ratio depending on the user's
                              # input value. E.g. the default `-1x368` is equivalent to `656x368` in 16:9 resolutions,
                              # e.g. full HD (1980x1080) and HD (1280x720) resolutions.
                              # for more fps AND LOWER ACCURACY use: "320x176"
falg_face: false             # Enables face keypoint detection. It will share some parameters from the body pose, e.g. model folder
flag_logging_level: 3        # Current OpenPose library messages are in the range 0-4: 1 for
                              # low priority messages and 4 for important ones, = will output any log message
flag_model_folder: "/home/novotnyg/git/Programs/openpose/"  # Folder path (absolute or relative) where the models (pose, face, ...) are located
flag_render_thrash: 0.05            # Only estimated keypoints whose score confidences are higher than this threshold will be
                                # rendered. Generally, a high threshold (> 0.5) will only render very clear body parts
                                # while small thresholds (~0.1) will also output guessed and occluded keypoints, but also
                                # more false positives
flag_disable_blending: false  # If blending is enabled, it will merge the results with the original frame. If disabled, it
                                  # will only display the results
flag_alpha_pose: 0.6         # Blending factor (range 0-1) for the body part rendering. 1 will show it completely, 0 will
                                # hide it. Only valid for GPU rendering.
flag_output_resolution: "-1x-1"  # "The image resolution (display and output). Use \"-1x-1\" to force the program to use the
                                      #input image resolution.
scale_number: 1              # Number of scales to average
scale_gap: 0.3               # Scale gap between scales. No effect unless num_scales>1. Initial scale is always 1. If you
                                    # want to change the initial scale, you actually want to multiply the `net_resolution` by
                                    # your desired initial scale.
no_display: false            # Do not open a display window 
#show_skeleton: true           # Flag that decides wether to publish output image or not
#pub_bbox: true                # Flag that decides wether to publish the Bounding Box image
